islandora.module

```diff
@@ -2170,3 +2170,23 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $object = islandora_object_load($cmodel);
+      $rows[] = array(l($object->label, "islandora/object/{$object->id}"));
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = theme('table', array(
+    'header' => array($header),
+    'rows' => $rows,
+    'empty' => t('No content model associated with this object.')));
```

Comment:
Somewhat of nitpicking, but this isn't _strictly_ true, because `fedora-system:FedoraObject-3.0`... That said, I'm not sure how exactly it might be phrased.